thefuck/rules/git_remote_seturl_add.py

```diff
@@ -0,0 +1,11 @@
+from thefuck.utils import replace_argument
+from thefuck.specific.git import git_support
+
+@git_support
+def match(command):
+    return('set-url' in command.script and 'fatal: No such remote' in command.stderr)
+
+def get_new_command(command):
```

Comment:
Same here :+1: You might already know but it's always worth mentioning Python's [Code Style](http://docs.python-guide.org/en/latest/writing/style/).